init.py

```diff
@@ -0,0 +1,57 @@
+import requests
+import zipfile
+import io
+import os
+import sys
+
+repos_url = "https://api.github.com/users/"
+repo_zip_url = "https://github.com/"
+
+def getRepos(username=None):
+    print ("Getting repository list")
+    try:
+        url = repos_url + username + "/repos"
+        response = requests.get(url)
+        repos = response.json()
+        return repos
+    except Exception as e:
+        print (e)
+   
+def saveRepos(username, repos=None):
+    if repos:
+        abspath = os.path.abspath(sys.path[0])
+        dname = os.path.dirname(abspath + '/bitnaughts/Assets/')
+        print (dname)
+        os.chdir(dname)
+        try:
+            repo_number = 1
+            for repo in repos:
+                reponame = repo["name"]
+                print (str(repo_number) + " " + reponame + " downloading....")
+                repozipurl = repo_zip_url + username + "/" + reponame + "/archive/master.zip"
```

Comment:
Similar to above, variable names seem to be inaccurate. Also if we could follow closer to the BitNaughts defined convention for naming, it'd help in code readability. (see https://github.com/bitnaughts/csharp.interpreter/blob/develop/README.md)